conventions/Documentation.md

```diff
@@ -1,20 +1,67 @@
 # Documentation
 
+## Stencil API reference
+
+Stencil uses [JSDoc](https://jsdoc.app) for their API reference generation. Stencil generates a [`docs-json`](https://stenciljs.com/docs/docs-json) output target, which is parsed and displayed on the [SDK site](https://developers.arcgis.com/calcite-design-system/components). The API reference includes property/attribute names, descriptions, types, values, and description notes (e.g. required, optional, deprecated). The SDK site updates the API reference after Calcite Component releases.
+
+### Style guide
+
+Follow these conventions when adding or editing API reference:
+
+- Use complete sentences with proper grammar, capitalization, and punctuation.
+- No abbreviations, e.g. use "property" instead of "prop".
+- Verbs must be in present tense.
+- Use the full tag name when referencing other Calcite Components (prefix with `calcite-`), e.g. `calcite-button` instead of `button`.
+- For plural context, use `calcite-button`s instead of `calcite-button` elements.
+- Use backticks (`` ` ``) for the names of slots, events, properties CSS variables, and component names (e.g. `calcite-button` instead of calcite-button).
+- Use double quotes (`"`) for the values of properties/attributes and event details.
+- Only use single quotes (`'`) as apostrophes.
+- No links or URLs allowed in descriptions. If a link is necessary, a [custom JSDoc tag](https://stenciljs.com/docs/docs-json#custom-jsdocs-tags) should be added and parsed in the SDK site.
+- Refrain from using "e.g." or "i.e." references. Leverage "such as" (or similar) where examples are referenced.
+- Use "Accessible" instead of "Aria" or "a11y" language
+- Verify slots and properties/attributes don't use the text "optional" in their descriptions
```

Comment:
Some ending sentences?:

- Use "Accessible" instead of "Aria" or "a11y" language[.]
- Verify slots and properties/attributes don't use the text "optional" in their descriptions[.]